lib/backend.js

```diff
@@ -377,36 +377,26 @@ exports.createServer = (dynamodb, docClient) => {
   app.get('/tables/:TableName', asyncMiddleware((req, res) => {
     const TableName = req.params.TableName
     req.query = pickBy(req.query)
-    const filters = omit(req.query, ['_hash', 'range', 'prevKey', 'startKey', 'pageNum'])
+    const filters = JSON.parse(req.query.filters || '{}')
 
     return describeTable({ TableName })
       .then(description => {
         const pageNum = req.query.pageNum ? parseInt(req.query.pageNum) : 1
-        const ExpressionAttributeNames = {}
-        const ExpressionAttributeValues = {}
-        const FilterExpressions = []
-
-        for (const key in filters) {
-          const attributeDefinition = description.Table.AttributeDefinitions.find(
-            definition => {
-              return definition.AttributeName === key
-            }
-          )
-          if (attributeDefinition && attributeDefinition.AttributeType === 'N') {
-            req.query[key] = Number(req.query[key])
-          }
-          ExpressionAttributeNames[`#${key}`] = key
-          ExpressionAttributeValues[`:${key}`] = req.query[key]
-
-          FilterExpressions.push(`#${key} = :${key}`)
-        }
 
         const data = Object.assign({}, description, {
           query: req.query,
           omit,
           filters,
-          pageNum: pageNum,
+          pageNum,
           filterQueryString: querystring.stringify(filters),
```

Comment:
There is a bug now with handling next/previous page links. They don't preserve filters anymore.

```
> JSON.stringify(filters)
< "{"destination":{"operator":"=","value":"template"}}"

> querystring.stringify(filters)
< "destination="
```